pkg/manager/internal.go

```diff
@@ -423,7 +413,7 @@ func (cm *controllerManager) serveHealthProbes(stop <-chan struct{}) {
 		Handler: mux,
 	}
 	// Run server
-	cm.startRunnable(RunnableFunc(func(stop <-chan struct{}) error {
+	cm.startRunnable(RunnableFunc(func(ctx context.Context) error {
```

Comment:
could we change this to `func(_ context.Context` to make it clearer, that the context we pass here is not used?